### PR TITLE
append suggestions dropdown in body

### PR DESCRIPTION
**Pull request by Psyphant**
see https://github.com/webscopeio/react-textarea-autocomplete/issues/128

### CODE REVIEW

**Comment by jukben** [comment]
(bot)

Hey! Thank you so much for your PR! I can see that everything is green 👏.

[Here is the playground for this revision](https://1011-94480675-gh.circle-artifacts.com/0/example/index.html) 🚀

I hope everything is fine! ❤️

**Comment by jukben** [comment]
Awesome, to see that `boundariesElement` works still like a charm. I would probably introduce another property `renderToBody` which does what you have here, otherwise I'll fallback to the original solution. What do you think about it? Are you willing to introduce the changes? 

thank you for raising this! 🚀

**Comment by Psyphant** [comment]
Hey @jukben ... the new property `renderToBody` is born.

**Comment by Psyphant** [comment]
Done 👍

**Comment by jukben** [comment]
Hey I'm testing https://1026-94480675-gh.circle-artifacts.com/0/example/index.html but it looks like left position is not updated correctly (for example when you typing). Are you able to confirm it?

File: src/Textarea.jsx [hunk 1]
```diff
@@ -152,8 +152,8 @@ class Autocomplete extends React.Component<AutocompleteProps> {
     }
 
     if(this.props.renderToBody) {
-      topPosition = topPosition + textareaBounds.top;
-      leftPosition = leftPosition + textareaBounds.left;
+      topPosition =+ textareaBounds.top;
```
Comment (Psyphant) [inline src/Textarea.jsx]: += instead of =+

**Comment by Psyphant** [comment]
Yeah ... my last change destroyed it 😅 
The solution was simple: "+=" instead of "=+" 🤦‍♂

**Comment by jukben** [comment]
Awesome, could you please add it also into docs (https://github.com/webscopeio/react-textarea-autocomplete#props). Right after `textAreaComponent` should be fine. Last thing and we are done! Thanks for the dope contribution! 🚀

File: package.json [hunk 2]
```diff
@@ -82,7 +82,8 @@
   },
   "peerDependencies": {
     "prop-types": "^15.0.0",
-    "react": "^16.0.0"
+    "react": "^16.0.0",
+    "react-dom": "16.6.0"
```
Comment (jukben) [inline package.json]: Haha, good catch!

**Pull request merged**